contrib/cf-info/cf-info.sh

```diff
@@ -28,9 +28,11 @@ echo
 echo "Host key:"
 /var/cfengine/bin/cf-key -p /var/cfengine/ppkeys/localhost.pub
 echo
-awk -F '[:,]' -v time="$(date +%s)" 'END {printf "Last cf-agent run started %d minutes ago and lasted %d seconds\n", (time - $1)/60, $2 - $1}' /var/cfengine/promise_summary.log
+grep -v update.cf  /var/cfengine/promise_summary.log | awk -F '[:,]' -v time="$(date +%s)" 'END {printf "Last cf-agent run started %d minutes ago and lasted %d seconds\n", (time - $1)/60, $2 - $1}'
```

Comment:
Can the regex matching be done in `awk` too?

Reply:
It's possible for there to be other entries, so if you are only interested in promises.cf, perhaps:

```
awk -F '/CFEngine Promises.cf/[:,]' -v time="$(date +%s)" 'END {printf "Last cf-agent run started %d minutes ago and lasted %d seconds\n", (time - $1)/60, $2 - $1}' /var/cfengine/promise_summary.log 
Last cf-agent run started 4 minutes ago and lasted -1554913420 seconds
```